CONTRIBUTING.md

```diff
@@ -1,81 +1,120 @@
-## Contribution Guide
+# Contribution Guide
 
-### Settting up your local development environment
+## Settting up your local development environment
 
-#### Add the wasm target
+### Add the wasm target
 
 ```bash
 rustup target add wasm32-unknown-unknown
 ```
 
-#### Build
+### Build
 
 ```bash
 cargo build --target wasm32-unknown-unknown
 ```
 
-#### Examples
+#### stdweb
 
-TODO: Add more info 
-
-##### stdweb
-In order to run the examples in `./yew-stdweb`, you may wish to install [cargo-web]:
+In order to run the examples in `./yew-stdweb`, you may wish to install [cargo-web](https://github.com/koute/cargo-web):
 
 ```bash
 cargo install cargo-web
 ```
 
-[cargo-web]: https://github.com/koute/cargo-web
+## Tests
 
-#### Test
+Yew has unit tests which can be run with `cargo test`. However, a large portion of the tests need to be run in a browser.
```

Comment:
```suggestion
Yew has unit tests which can be run with `cargo test`. However, a large portion of these tests need to be run in a browser.
```

Reply:
Wouldn't using "these" imply that we're talking about a subset of the unit tests?
I'm not trying to be pedantic on nomenclature but the browser tests aren't unit tests, they're integration tests.